app/controllers/api/v1/dashboard_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Api::V1::DashboardController < Api::V1::AuthorizationController
   def index
-    cocktails = Cocktail.dashboard_five
+    cocktails = Cocktail.dashboard_five(current_user.id)
```

Comment:
Thinking aloud... does this already check whether the user who just signed up is a new user? or is that done on the FE? Just was thinking and wasn't sure in my mind...

Reply:
@markcyen the FE sends you to a different page if you're not a new user I believe. We def are not currently checking that on the BE